requirements/main.txt

```diff
@@ -1,8 +1,8 @@
 certifi==2020.4.5.1
 chardet==3.0.4
-cnx-common==1.2.3
+cnx-common==1.3.3
 cnx-db==3.5.3
-cnx-epub==0.21.0
+cnx-epub==0.21.1
```

Comment:
pin to lxml will also have to change to 4.4.3 because of this

Reply:
Ah thanks!